packages/boxel-ui/addon/src/components/resizable-panel-group/panel.gts

```diff
@@ -4,24 +4,26 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 
 import cssVars from '../../helpers/css-var.ts';
+import { eq } from '../../helpers/truth-helpers.ts';
 
 export type PanelContext = {
-  defaultWidth: string;
-  minWidth?: string;
-  width: string;
+  defaultLength: string;
+  length: string;
+  minLength?: string;
 };
 
 interface Signature {
   Args: {
-    defaultWidth: string;
+    defaultLength: string;
     isLastPanel: (panelId: number) => boolean;
-    minWidth?: string;
+    length?: string;
+    minLength?: string;
     onResizeHandlerDblClick: (event: MouseEvent) => void;
     onResizeHandlerMouseDown: (event: MouseEvent) => void;
+    orientation: 'horizontal' | 'vertical';
     panelContext: (panelId: number) => PanelContext | undefined;
-    // The following arguments will be supplied by the parent ResizablePanelGroup that yields this component
```

Comment:
This comment was incorrect because the interface properties are now alphabetically sorted.